### PR TITLE
RO substitution of SLA files to MD in QuickFacts

### PR DESCRIPTION
**Pull request by BlackLotus64**
Another substitution from the SLA files to MDs, this time is from QuickFacts which is the only file in the language.

@lh1008

### CODE REVIEW

**Comment by BlackLotus64** [comment]
Solved @erciccione, sorry, it happens sometimes when I open the file in Scribbus.

File: monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+**_[getmonero.org (Official Website)](https://getmonero.org/)_**
+**_[monero.how](https://www.monero.how/)_**
+**_[reddit.com/r/monero](https://www.reddit.com/r/Monero/)_**
+**_[Guide to Monero (post)](https://www.reddit.com/r/CryptoCurrency/comments/7ra409/your_guide_to_monero_and_why_it_has_great/)_**
+**_[Monero In-Depth Technical Intro](https://steemit.com/monero/@sgp/7yjqso-a-monero-introduction-for-beginners)_**
+**_[Zero to Monero](https://www.getmonero.org/library/Zero-to-Monero-1-0-0.pdf)_**
+
+**_[Scams to Avoid](https://www.reddit.com/r/Monero/wiki/avoid)_**
+**_[Monero FAQ](https://ww.getmonero.org/get-started/faq/)_**
+**_[Monero SWOT Analysis]()_**
```
Comment (erciccione) [inline monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: Why is this link empty? @lh1008 ?

File: monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+**_[Connect w/ Monero Community](https://getmonero.org/community/hangouts/)_**
+**_[Mastering Monero - (coming soon)](https://masteringmonero.com/)_**
```
Comment (erciccione) [inline monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]:
```suggestion
**_[Mastering Monero](https://masteringmonero.com/)_**
```

File: monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md (hunk 1 above)
Comment (BlackLotus64) [reply monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: It's empty in all the other files too, and PDFs.

File: monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md (hunk 2 above)
Comment (BlackLotus64) [reply monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: So it came out, nice.

File: monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md (hunk 1 above)
Comment (erciccione) [reply monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: Ok, got it.

Comment (lh1008) [reply monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: We had a SWOT analysis we found in the web, but suddenly it disappeared from the web. So we deleted the link and are currently working on the SWOT analysis directly from the Monero Outreach workgroup through taiga [#66 SWOT Analysis](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/us/66). :)

File: monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md (hunk 2 above)
Comment (lh1008) [reply monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: Great! :)

File: monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
+# MONERO
+
+Monero este o valută digitală securizată,
+confidențială si nedetectabilă care este
+disponibilă tuturor.
+
+# SUMAR
+
+Utilizatorii pot cheltui in sigurantă, nimeni nu poate vedea soldul sau să urmarească tranzacțiile.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: There is a strange thing here.

Comment (BlackLotus64) [reply monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: This happens when I directly copy and paste from scribbus and the text has various lines like that part, but I'm not sure why.

**Comment by BlackLotus64** [comment]
Changes made, ready to merge/review :)

File: monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+**Minatul este accesibil:** Oricine are un dispozitiv conectat sau un browser web poate participa.
+
+### IMPLICATII SI UTILIZARI IN LUMEA REALA
+
+Deoarece Monero este sigur, cu taxe reduse si fara frontiere, oamenii pot trimite cu usurinta bani in ciuda guvernelor corupte sau bancilor. Aceasta ofera imputernicire economica persoanelor in **[tari opresive](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)**.
+
+Istoria finaciara privata protejeaza consumatorii si companiile de manipularile de pret, exploatarea lantului de aprovizionare, discriminare economica sau altele. **[Monero este singura criptomoneda](https://www.reddit.com/r/Monero/comments/8k8pk9/monero_the_worlds_bestkept_secret/)** care are caracteristici necesare pentru a servi ca bani complet fungibili, descentralizati si electronici.
+
+### FUNDAMENTALE TEHNICE
```
Comment (erciccione) [inline monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: The format of this section is completely broken

File: monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md (hunk 3 above)
Comment (erciccione) [reply monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md]: > I directly copy and paste from scribbus

This is not how a migration to a different format should be done. Markdown has its own specific structure, which is not compatible with scribus'. Please make sure to export the text in the correct formatting and double check the resulting text or the format will break. See [#78 (comment)](https://github.com/monero-ecosystem/outreach-docs/pull/78/files#r248334206)

**Comment by erciccione** [comment]
After a quick check i realized that all markdown documents recently PRd by @BlackLotus64 are broken in the format. I just opened #79

**Comment by BlackLotus64** [comment]
Updated from #80 and solving #79

**Pull request merged**